lib/uwb/UwbPeer.cxx

```diff
@@ -32,6 +52,17 @@ UwbPeer::operator=(const UwbPeer& other)
     return *this;
 }
 
+std::string
+UwbPeer::ToString() const
+{
+    std::stringstream ss;
+    ss << "{" << std::endl;
+    ss << "mac: " << m_address.ToString() << std::endl;
+    ss << m_spatialProperties.ToString() << std::endl;
```

Comment:
If you include `<notstd/tostring.hxx>` and do a `using namespace strings::ostream_operators`, you can just pass the object directly, and it will invoke the `ToString()` method automatically:

```suggestion
    ss << "mac: " << m_address << std::endl;
    ss << m_spatialProperties << std::endl;
```

Reply:
Since this doesn't seem to be working in this instance, let's skip it for now.